tests/transfomers/portable-text-transformer/resolution/vue.resolver.spec.ts

```diff
@@ -2,12 +2,12 @@ import { PortableText, PortableTextComponentProps, PortableTextComponents, toPla
 import { mount } from "@vue/test-utils";
 import { h } from "vue";
 
-import { nodeParse, PortableTextImage, PortableTextTable, transformToPortableText } from "../../../src";
-import { resolveImage, resolveTable, toVueImageDefault } from "../../../src/utils/resolution/vue";
+import { PortableTextImage, PortableTextTable, transformToPortableText } from "../../../../src";
+import { resolveImage, resolveTable } from "../../../../src/utils/resolution/vue";
 
 const components: PortableTextComponents = {
   types: {
-    image: ({ value }: PortableTextComponentProps<PortableTextImage>) => resolveImage(value, h, toVueImageDefault),
+    image: ({ value }: PortableTextComponentProps<PortableTextImage>) => resolveImage(value, h),
     table: ({ value }: PortableTextComponentProps<PortableTextTable>) => resolveTable(value, h, toPlainText),
```

Comment:
@pokornyd not sure if this was changed, but when testing it I noticed that all properties under `types` were required - meaning you had to provide some sort of implementation even if you only needed to provide a resolver for image. Can these props be made optional instead?

Reply:
the helper types (`PortableTextHtmlResolvers`, `PortableTextReactResolvers`) are both set to be partial. I didn't create helper type for Vue, but `PortableTextComponents["types"]` appears to be partial by default